meetings/2023-06-07.md

```diff
@@ -0,0 +1,74 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2023-06-07
+
+## Links
+
+* **Recording**:  <https://www.youtube.com/watch?v=LZxU9WaZ26Y>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1396>
+
+## Present
+
+* Antoine du Hamel @aduh95 (voting member)
+* Geoffrey Booth @GeoffreyBooth (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Moshe Atlow @MoLow (voting member)
+* Rafael Gonzaga @RafaelGSS (voting member)
+* Robert Nagy @ronag (voting member)
+* Ruy Adorno @ruyadorno (voting member)
+
+## Agenda
+
+### Announcements
+
+* Rafael - Node.js 20.3 should be released tomorrow
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* No updates.
+
+### nodejs/node-gyp
+
+* DISCUSSION: Release handling [#2860](https://github.com/nodejs/node-gyp/issues/2860)
+  * Michael gave a quick overview of the issue
+  * Rafael, one idea might be to ask Foundation to fund rewriting in Node.js
+  * Need to ask more broadly if we have any collaborators that have skills/would be interested in getting involved.
+
+### nodejs/node
+
+* module: make CJS load from ESM loader [#47999](https://github.com/nodejs/node/pull/47999)
+  * Added to agenda for visibility, not much to discuss
+  * Related Geoffrey, Qard added for more guidance on handling breaking changes on experimental features - <https://github.com/nodejs/TSC/issues/1397>
+
+### nodejs/TSC
+
+* CLI tools behind a Nodejs namespace [#1178](https://github.com/nodejs/TSC/issues/1178)
+  * leave on agenda until we have Daniele to confirm we have covered
+  * ok to start migrating packages to the node-core namespace.
+  * Rafael will add comment on 1178 that covers how it was handled with fastify.
+    * <https://github.com/fastify/deprecate-modules>
+    * Workflow assuming package is currently on version v2.x:
+      * Release a new major with npm deprecate pointing to the new module and no content, so version v3.0.0
+      * Create a new package with the correct name: @node-core/module in the version v4.0.0
+
+### nodejs/admin
+
+* Archive nodejs/help in favour of a Help Section on Discussions [#802](https://github.com/nodejs/admin/issues/802)
+  * Lets defer until a future meeting where ideally we have Gireesh
+
+## Strategic Initiatives
+
+* Yagiz - Recent changes from the performance initiative:
+  * I opened a back-port for Ada v2.5 to LTS (which comes with 100% performance boost)
```

Comment:
FWIW, I know that this is just quoting another comment verbatim, but we should be careful to make accurate statements about performance improvements. Are _all_ `URL` operations really twice as fast now on all platforms, especially as visible from JS? Maybe, I have no idea, but it has happened before that performance initiatives misrepresented or generalized results of microbenchmarks.

Reply:
@anonrig can you take a look and suggest updated text for this one and the later comment as well?

Reply:
I'm currently on paternity leave but we can remove all numeric values, and use terms like "significant improvements", "major changes" etc. if not, we can just reduce the changes to "improved the performance of X"

Reply:
```suggestion
  * I opened a back-port for Ada v2.5 to LTS (which comes with performance improvements)
```